lib/experiments-page.php

```diff
@@ -199,6 +199,19 @@ function gutenberg_initialize_experiments_settings() {
 		)
 	);
 
+	add_settings_field(
+		'gutenberg-zoom-out-experiment',
+		__( 'Zoom out experiments', 'gutenberg' ),
+		'gutenberg_display_experiment_field',
+		'gutenberg-experiments',
+		'gutenberg_experiments_section',
+		array(
+			'label' => __( 'Enable zoom out experiments; shows zoom out in the device preview and other zoom out experiments.', 'gutenberg' ),
```

Comment:
The plural here encouraged me to believe this would deactivate all Zoom Out stuff when unchecked. However, if I have the Zoom Out in Patterns experiment enabled it still goes into Zoom Out.

If what we're doing is hiding the Zoom Out in Device Preview _only_ then we should update the text to reflect that.

Also can we group this Zoom Out setitng with the other one (or just place next to it)?

Reply:
I guess https://github.com/WordPress/gutenberg/pull/65045 answers my question?

Reply:
I think we should use this experiment for other zoom out explorations, including the new page flow, which is why the wording is somewhat generic.

Reply:
Yes I agree. It should be a one stop shop for all Zoom Out experiments for now. If we need to get fine-grained we can do that later.